cmake/CMakeLists_googletest_download.txt.in

```diff
@@ -8,18 +8,22 @@ project(googletest-download NONE)
 # https://github.com/abseil/googletest/blob/master/googletest/README.md
 # need to force the use of the shared C run-time.
 
-
 include(ExternalProject)
 
 set(FORCE_SHARED_CRT "")
 if(MSVC)
   set(FORCE_SHARED_CRT -DFORCE_SHARED_CRT=OFF)
 endif()
 
+set(OSX_RPATH "")
+if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+  set(OSX_RPATH -DCMAKE_MACOSX_RPATH=1)
+endif()
+
 ExternalProject_Add(googletest
   GIT_REPOSITORY    https://github.com/google/googletest.git
   GIT_TAG           master
   SOURCE_DIR        "${GOOGLETEST_BUILD_ROOT}/googletest-src"
   BINARY_DIR        "${GOOGLETEST_BUILD_ROOT}/googletest-build"
-  CMAKE_ARGS        "-DCMAKE_MAKE_PROGRAM=${CMAKE_MAKE_PROGRAM}" "-DCMAKE_INSTALL_PREFIX=${GOOGLETEST_BUILD_ROOT}/googletest-install" "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}" "-Dgtest_force_shared_crt=ON" "-DCMAKE_POSITION_INDEPENDENT_CODE=ON" -DFORCE_SHARED_CRT=OFF ${FORCE_SHARED_CRT}
+  CMAKE_ARGS        "-DCMAKE_MAKE_PROGRAM=${CMAKE_MAKE_PROGRAM}" "-DCMAKE_INSTALL_PREFIX=${GOOGLETEST_BUILD_ROOT}/googletest-install" "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}" "-Dgtest_force_shared_crt=ON" "-DBUILD_SHARED_LIBS=1" "-DCMAKE_MACOSX_RPATH=1" "${OSX_RPATH}" "-DCMAKE_POSITION_INDEPENDENT_CODE=ON" ${FORCE_SHARED_CRT}
```

Comment:
Building GTest as shared library. To use rpaths on macOS, CMAKE_MACOSX_RPATH needs to be set. 

We had to switch from static library to shared library because Gtest has a global container which holds all registered tests and registration of tests happens during Maya plugin load, execution of tests happens from utility library (via Maya's command). When plugin and utility library were linking with GTest statically, on windows, tests were registered in one container which wasn't shared with utility library, causing no tests being executed.

Reply:
Totally a typo on my part!!! See 26e38348b13 for fix.